### PR TITLE
Features/bump r2d2 to 0 8 2

### PR DESCRIPTION
**Pull request by ernestas-poskus**
@AlexPikalov 

Hey, Happy upcoming NY, bumping r2d2 version and fixed `is_alive` function return value.

### CODE REVIEW

**Comment by ernestas-poskus** [comment]
Windows build fails with some sort of unknown argument for nightly

File: src/transport.rs [hunk 1]
```diff
@@ -201,6 +201,6 @@ impl CDRSTransport for TransportTls {
     }
 
     fn is_alive(&self) -> bool {
-        self.ssl.get_ref().peer_addr().is_ok();
+        self.ssl.get_ref().peer_addr().is_ok()
```
Comment (AlexPikalov) [inline src/transport.rs]: Good catch!

**Comment by ernestas-poskus** [comment]
Weird now Appveyor passed (haven't done anything)

**Comment by AlexPikalov** [comment]
Yup! Merging.

**Pull request merged**